sdk/keyvault/Azure.ResourceManager.KeyVault/samples/Generated/Samples/Sample_SubscriptionResourceExtensions.cs

```diff
@@ -187,7 +187,7 @@ public async Task CheckManagedHsmNameAvailability_ValidateAManagedHsmName()
 
             // invoke the operation
             ManagedHsmNameAvailabilityContent content = new ManagedHsmNameAvailabilityContent("sample-mhsm");
-            ManagedHsmNameAvailabilityResult result = await subscriptionResource.CheckManagedHsmNameAvailabilityAsync(content);
+            ManagedHsmNameAvailabilityResult result = await subscriptionResource.CheckManagedHsmNameAvailabilityManagedHsmAsync(content);
```

Comment:
mapping failed: https://github.com/Azure/azure-sdk-for-net/blob/91af14e56d8b9f12d66fc5217e2e13670a21d8e1/sdk/keyvault/Azure.ResourceManager.KeyVault/src/autorest.md?plain=1#L26